pkg/mount/dag_uki_boot.go

```diff
@@ -20,7 +20,7 @@ func (s *State) RegisterUKI(g *herd.Graph) error {
 	s.LogIfError(s.UKIUdevDaemon(g), "udev")
 
 	// Run rootfs stage
-	s.LogIfError(s.RootfsStageDagStep(g, cnst.OpSentinel, cnst.OpUkiUdev), "uki rootfs")
+	s.LogIfError(s.RootfsStageDagStep(g, herd.WithDeps(cnst.OpSentinel, cnst.OpUkiUdev), herd.WithWeakDeps()), "uki rootfs")
```

Comment:
empty weakdeps?

Reply:
yes, signature of the call has changed to be able to pass an optional number of deps and weakdeps